guby_backend/views.py

```diff
@@ -48,6 +53,10 @@ def meetup_add(request):
          meetup = form.save(commit=False)  
          meetup.creator = request.user
          meetup.save()
+
+         # create group to handle meetup ownership
+         owner_group, created = Group.objects.get_or_create(name=f'guby-meetup-{meetup.id}-owners')
```

Comment:
This looks good 👍

Reply:
considering we're doing a [split operation](https://github.com/unnati-xyz/guby/pull/7/files/dea54cc007d1c28f6f287d8762d950d83ee0ae04..6db3717181afb69639ba2c33ab04c72636da1f78#diff-9e43c299323a4a92a543ea12ea1bed3cR50) to get the meetup ID, do you think it will be better to keep it in a easy to parse state instead?

Something like `guby-meetup-owners#{meetup.id}`. That way:

- Meetup ID is always at the end
- Having a unique character `#` to split on, thereby not relying on any of us accidentally adding another `-` in there and screwing the entire parse operation up

Thoughts?

Reply:
@shrayasr  changed the group name to `meetup-owner#{meetup.id}`